apps/src/templates/sectionAssessments/sectionAssessmentsRedux.js

```diff
@@ -699,13 +702,15 @@ export const getStudentsMCSummaryForCurrentAssessment = state => {
  */
 export const getExportableSubmissionStatusData = state => {
   let summaryStudentStatus = [];
-  const studentStatus = getStudentsMCSummaryForCurrentAssessment(state);
+  const studentStatus = getStudentsMCandMatchSummaryForCurrentAssessment(state);
 
   studentStatus.forEach(student => {
     summaryStudentStatus.push({
       studentName: student.name,
       numMultipleChoiceCorrect: student.numMultipleChoiceCorrect,
       numMultipleChoice: student.numMultipleChoice,
+      numMatchCorrect: student.numMatchCorrect,
+      numMatch: student.numMatch,
```

Comment:
Is there a reason to combine getting the match and multiple choice rather than getting the multiple choice, and separately getting the match? No issues with how you've implemented, just curious.

Reply:
We use this function to fill in a table that shows the summary of the whole assessment so both sets of numbers are used in that table.